src/pybind/mgr/restful/module.py

```diff
@@ -244,18 +245,43 @@ def _serve(self):
             separators=(',', ': '),
         )
 
-        cert = self.get_config_json("cert") or '/etc/ceph/ceph-mgr-restful.crt'
-        pkey = self.get_config_json("pkey") or '/etc/ceph/ceph-mgr-restful.key'
+        server_addr = self.get_config('server_addr') or '127.0.0.1'
+        server_port = int(self.get_config('server_port') or '8003')
+        self.log.info('server_addr: %s server_port: %d',
+                      server_addr, server_port)
+
+        cert_fname = None
+        pkey_fname = None
```

Comment:
There is no need to initialize variables like this in python, they will be overwritten (re-initialized) by next assignment anyway (that holds even for if/else like you use later in the code). The same goes for cert_tmp and pkey_tmp.